Makefile

```diff
@@ -415,3 +415,16 @@ test-app-dev: tmp/.ekam-run
 	@cp src/sandstorm/test-app/test-app.capnp tmp/sandstorm/test-app/test-app.capnp
 	@cp src/sandstorm/test-app/*.html tmp/sandstorm/test-app
 	spk dev -Isrc -Itmp -ptmp/sandstorm/test-app/test-app.capnp:pkgdef
+
+# ====================================================================
+# meteor-testapp.spk
+
+meteor-spk-0.3.2/meteor-spk:
+	@$(call color,downloading meteor-spk)
+	@curl https://dl.sandstorm.io/meteor-spk-0.3.2.tar.xz | tar Jxf -
+
+tests/assets/meteor-testapp.spk: meteor-testapp meteor-spk-0.3.2/meteor-spk $(shell find meteor-testapp -type f -o -type d)
+	@cd meteor-testapp && PATH="$$PATH:$$PWD/bin" && ../meteor-spk-0.3.2/meteor-spk pack ../tests/assets/meteor-testapp.spk
```

Comment:
When I prepend sandstorm/bin to PATH as written above, I get an error when trying to build meteor-testapp.spk:

>*** Uncaught exception ***
>sandstorm-pkgdef.capnp:2: failed: Import failed: /sandstorm/package.capnp
>Makefile:430: recipe for target 'tests/assets/meteor-testapp.spk' failed
>make: *** [tests/assets/meteor-testapp.spk] Error 1

Is this because there is a sandstorm binary inside sandstorm/bin that is the first sandstorm encountered in PATH?